packages/clerk-js/src/ui/Components.tsx

```diff
@@ -128,7 +128,7 @@ export const mountComponentRenderer = (clerk: Clerk, environment: EnvironmentRes
           void preloadComponent(preloadHint);
         }
         componentsControlsResolver = import('./lazyModules/common').then(({ createRoot }) => {
-          createRoot(clerkRoot!).render(
```

Comment:
Is this changed by mistake from the linter or it's a conscious change?

Reply:
Linter changed it.

Reply:
We should revert it then